HotSpot/AppDelegate.m

```diff
@@ -7,24 +7,83 @@
 //
 
 #import "AppDelegate.h"
-#import "Parse/Parse.h"
+#import "DataManager.h"
+#import "Booking.h"
 
 @interface AppDelegate ()
 
 @end
 
 @implementation AppDelegate
 
+PFUser *homeowner;
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(  NSDictionary *)launchOptions {
     
-    ParseClientConfiguration *config = [ParseClientConfiguration   configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
-        
-        configuration.applicationId = @"myAppId";
-        configuration.server = @"http://hotspot2017.herokuapp.com/parse";
+    [DataManager configureParse];
+    
+    [DataManager getHomeownersWithCompletion:^(NSArray<PFUser *> * _Nonnull homeowners, NSError * _Nonnull error) {
+        NSLog(@"%@", homeowners);
+        homeowner = homeowners[0];
+    }];
+    
+    // Fake a login
+    
+    NSString *username = @"user1";
+    NSString *password = @"password";
+    
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        if (error != nil) {
+            NSLog(@"User log in failed: %@", error.localizedDescription);
+        } else {
+            // success
+        }
+    }];
+    
+    [Booking getBookingsWithBlock:^(NSArray<Booking *> * _Nonnull bookings, NSError * _Nonnull error) {
+        NSLog(@"%@", bookings);
     }];
     
-    [Parse initializeWithConfiguration:config];
+//    [Booking bookDriveway:homeowner
+//           withCompletion:nil];
+    
+    
+    // Fake some data for the homeowners
+    
+//    // initialize a user object
+//    PFUser *newHomeowner = [PFUser user];
+//
+//    // set user properties
+//    newHomeowner.username = @"homeowner1";
+//    newHomeowner.password = @"password";
+//    newHomeowner[@"address"] = @"1 hacker way";
+//
+//    // call sign up function on the object
+//    [newHomeowner signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
+//        if (error != nil) {
+//            NSLog(@"Error: %@", error.localizedDescription);
+//        } else {
+//            // all good
+//        }
+//    }];
+    
+    // Fake some data for the users
+    
+//    // initialize a user object
+//    PFUser *newUser = [PFUser user];
+//
+//    // set user properties
+//    newUser.username = @"user1";
+//    newUser.password = @"password";
+//
+//    // call sign up function on the object
+//    [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
+//        if (error != nil) {
+//            NSLog(@"Error: %@", error.localizedDescription);
+//        } else {
+//            // all good
+//        }
+//    }];
```

Comment:
remove commented our lines! we should be faking the data in our database, not in the code.